<launch>

<!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->

    <!--Run move_base_seqs node-->
        <node pkg="sim_localization" type="move.py" name="movebase_client_py" output="screen">
        
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <rosparam param="p_seq">[-1,-2,0,1,-2,0,0.5,0.5,0]</rosparam>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="qua_seq">[0,0,0,1,0,0,0.5,0.9,0,0,0.702,0.71]</rosparam>

        </node>

</launch>
